src/data-loader.test.ts

```
@@ -1804,3 +1805,356 @@ describe('calculateCostForEntry', () => {
 		});
 	});
 });
+
+describe('loadSessionWindowData', () => {
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)